### ISSUE TITLE
System partition taking 59.8 Gb 

### ISSUE THREAD

**Issue by ghost** [sebanc/brunch#1766]
So I installed brunch on a external drive, and the chrome system partition is taking up most of the space. A little amount can actually be used by files. I even tried the resize-data command and it did nothing. What can I do to fix this?
![Screenshot 2023-01-25 5 05 44 PM](https://user-images.githubusercontent.com/69481843/214712889-c92cff1c-bb9d-4624-9635-ad5f119e82ca.png)

**Comment by knguyen1401** [comment]
Yeah I'm also experiencing this. I'm seeing my "System" storage is increasing lately.

**Comment by Bubbles78959** [comment]
Oh I forgot to add, the hard drive is 60 GB, so the amount the system takes makes no sense with how much is allocated for other files-

**Comment by Bubbles78959** [comment]
So I just installed it on another hard drive which is 1 TB, and the system takes 153 GB... Is it just hogging free space or is that just how much chrome uses?
![Screenshot 2023-01-29 12 38 10 AM](https://user-images.githubusercontent.com/69481843/215309689-ea77e310-342f-49b9-bef8-4b03350a290d.png)

**Comment by knguyen1401** [comment]
Can you keep track of the system storage size?

**Comment by Cateners** [comment]
I have the same issue. I installed brunch on a 512GB hard drive, and the system shows that it occupies about 509GB of space in the settings. However, when I switched to the Linux system to check the disk status, I found that there was approximately 470GB of unallocated space. So, I directly appended it to the last partition. Now, the settings show that 77GB of space has been used. Although it's still a bit much, the remaining space is sufficient.
